sections/components.tex

```diff
@@ -35,7 +35,7 @@ \subsubsection{Alert Production}\label{sec:AP}
 AP may also be able to process non-standard visits with longer or shorter exposure times provided that, e.g., the visit image can be successfully PSF-matched and differenced with a template image.
 Alert generation in crowded fields may produce more than the maximum of 10,000 alerts per visit required to be supported by DM; alert generation in this circumstance is still under study.
 }.
-Due to the need for Data Release Production-derived templates, Alert Production cannot run at full scale and full fidelity during commissioning nor the first year of operations.  LSST DM is currently investigating options for Alert Production in year one.
+Due to the need for Data Release Production-derived templates, Alert Production cannot run at full scale and full fidelity during commissioning nor the first year of operations.  LSST DM is currently investigating options for Alert Production in year one \newtext{\citedsp{DMTN-107}}.
```

Comment:
\newtext{\citedsp{LSE-459, DMTN-107}}.  
(LSE-459 is the new policy doc in draft)

Reply:
I put this in but just as text for now; will update once the document is available somewhere people can find it.